### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by vijaytembugade**
- Added filters by 
     - Creator
     - author
     - category

### CODE REVIEW

**Review by Shrinkhlarajpoot — APPROVED** [review]

**Review by preethasri — APPROVED** [review]
good work

File: src/Components/Filters/Filters.js [hunk 1]
```diff
@@ -1,14 +1,49 @@
...
+const Filters = ({ setShowFilter }) => {
+  const { state, dispatch, allAuthors, allCategories, allCreator } =
+    useFilter();
+
+  const filterbarRef = useRef();
+
+  useOnClickOutside(filterbarRef, () => setShowFilter(false));
+
+  const [selectedAuthors, setSelectedAuthor] = useState(
+    state.byAuthor.map((value) => ({ value: value, label: value }))
```
Comment (t007rushi) [inline src/Components/Filters/Filters.js:18]: initial set value be variable fas per the convention you can assign manipulations it in variable

File: src/Contexts/filterContext/filterContext.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+      case BY_CREATOR: {
+        return { ...state, byCreator: action.payload };
+      }
+    }
+  };
+
+  const [state, dispatch] = useReducer(filterReducer, initialState);
+
+  console.log(state);
```
Comment (t007rushi) [inline src/Contexts/filterContext/filterContext.js:48]: remove console here

**Review by t007rushi — APPROVED** [review]
Great Work Thala 
Keep It up👍💯

**Pull request merged**